lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -9,7 +9,12 @@ def fn_name(): return sys._getframe(1).f_code.co_name
 try:
     import gi
 except ImportError:
-    raise ImportError("Gtk3 backend requires pygobject to be installed.")
+    try:
+        import pgi
+        pgi.install_as_gi()
```

Comment:
Yup, my thoughts exactly.  I did it this way because pgi still seems like an alpha, i.e. with lots of features missing that python-gi has, but they claim that they have everything working that they put in examples, and as they put matplotlib in an example, I guess it all works for us :).

Alternatively I have just thought about adding a ``preferred_gi`` rcParam which defaults to ``gi`` but can get set to ``pgi``, to decide which one exists as the fallback.  That allows people to easily try pgi even if they ``python-gi`` installed.